seedtest.cc

```diff
@@ -324,7 +324,7 @@ void buildHitTripletsCurve(const std::vector<HitVec>& evt_lay_hits, const BinInf
     const auto phiBinMinus = getPhiPartition(negPhi);
     const auto phiBinPlus  = getPhiPartition(posPhi);
 
-#ifdef DEBUG
+#ifdef BROKEN_DEBUG
     const float lay2phi = evt_lay_hits[2][ev.simTracks_[ev.simHitsInfo_[hit0.mcHitID()].mcTrackID()].getHitIdx(2)].phi();
     dprint("lay0 phi: " << hit0.phi() << " lay1 phi: " << hit1.phi() << std::endl <<
 	   "negPhi: " << negPhi << " lay2 phi: " << lay2phi << " posPhi: " << posPhi << std::endl <<
```

Comment:
It uses ev which isn't in scope, probably hasn't been compiled since some
reorganization that moved it to where ev isn't available:

seedtest.cc:328:43: error: use of undeclared identifier 'ev'
    const float lay2phi = evt_lay_hits[2][ev.simTracks_[ev.simHitsInfo_[...
                                          ^
seedtest.cc:328:57: error: use of undeclared identifier 'ev'
    const float lay2phi = evt_lay_hits[2][ev.simTracks_[ev.simHitsInfo_[...



> On Apr 30, 2016, at 07:50, Kevin McDermott <notifications@github.com> wrote:
> 
> In seedtest.cc:
> 
> > @@ -324,7 +324,7 @@ void buildHitTripletsCurve(const std::vector<HitVec>& evt_lay_hits, const BinInf
> >      const auto phiBinMinus = getPhiPartition(negPhi);
> >      const auto phiBinPlus  = getPhiPartition(posPhi);
> >  
> > -#ifdef DEBUG
> > +#ifdef BROKEN_DEBUG
> >      const float lay2phi = evt_lay_hits[2][ev.simTracks_[ev.simHitsInfo_[hit0.mcHitID()].mcTrackID()].getHitIdx(2)].phi();
> >      dprint("lay0 phi: " << hit0.phi() << " lay1 phi: " << hit1.phi() << std::endl <<
> >  	   "negPhi: " << negPhi << " lay2 phi: " << lay2phi << " posPhi: " << posPhi << std::endl <<
> 
> @dan131riley What about this particular section is broken? I presume the BROKEN_DEBUG implies something is non-standard with how you have reorganized debugging. I can fix it -- let me know!
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>